.env.sample

```diff
@@ -48,16 +48,16 @@ OPENSEARCH_MODEL_ID=
 #######################################################
 
 # lnd
-LND_CERT=<YOUR LND HEX CERT>
-LND_MACAROON=<YOUR LND HEX MACAROON>
-LND_SOCKET=<YOUR LND GRPC HOST>:<YOUR LND GRPC PORT>
+LND_CERT=
+LND_MACAROON=
+LND_SOCKET=
 
 # lnurl
-LNAUTH_URL=<PUBLIC URL TO /api/lnauth>
-LNWITH_URL=<PUBLIC URL TO /api/lnwith>
+LNAUTH_URL=
+LNWITH_URL=
 
 # nostr (NIP-57 zap receipts)
-NOSTR_PRIVATE_KEY=<YOUR NOSTR PRIVATE KEY IN HEX>
+NOSTR_PRIVATE_KEY=
```

Comment:
This is only required for working with nostr zaps but we could generate a random nostr private key in hex format via `openssl rand -hex 32` during runtime for dev builds.

Or fail in a less spectacular and more developer friendly way if this is not set but the code expects it.

Reply:
I preset it with a value for now